x-pack/plugins/security_solution/public/resolver/lib/date.ts

```diff
@@ -34,12 +34,12 @@ export const getUnixTime = (time: number | string): number | typeof NaN => {
 export const getFriendlyElapsedTime = (
   from: number | string,
   to: number | string
-): DurationDetails | null => {
+): DurationDetails | undefined => {
   const startTime = getUnixTime(from);
   const endTime = getUnixTime(to);
 
   if (Number.isNaN(startTime) || Number.isNaN(endTime)) {
-    return null;
+    return undefined;
```

Comment:
:grey_question: May want to update the doc comment above to reflect this behavior (e.g. "when start or end time are invalid, it returns undefined")?

Reply:
will do